pliers/config.py

```diff
@@ -25,7 +25,8 @@
     'n_jobs': None,
     'parallelize': False,
     'progress_bar': True,
-    'use_generators': False
+    'use_generators': False,
+    'allow_large_jobs': True
```

Comment:
We should probably add config keys that define what a large job is here (instead of hardcoding to 100 and 1 minute in the `Transformer` logic